src/controllers/apiController.ts

```diff
@@ -341,11 +341,11 @@ export class ApiController extends BaseController {
    *
    * @returns BXML string
    */
-  static generateTransferBxml(deviceToken: string, sipUri='sip:sipx.webrtc.bandwidth.com:5060'): string {
-    return '<?xml version="1.0" encoding="UTF-8" ?>\n'
-        + '<Response>'
-        + this.generateTransferBxmlVerb(deviceToken, sipUri)
-        + '</Response>';
+  static generateTransferBxml(deviceToken: string, sipUri='sip:sipx.webrtc.bandwidth.com:5060', voiceCallId: string): string {
```

Comment:
Add doc for the parameter into the function doc

Reply:
Good suggestion!